spot-client/src/common/app-state/api/middleware.js

```diff
@@ -0,0 +1,48 @@
+import { MiddlewareRegistry } from 'common/redux';
+
+import { logger } from '../../logger';
+
+import { SEND_API_MESSAGE } from './actionTypes';
+
+MiddlewareRegistry.register(() => next => action => {
+    const result = next(action);
+
+    switch (action.type) {
+    case SEND_API_MESSAGE:
+        _sendApiMessage(action.messageType, action.messageData);
+        break;
+    }
+
+    return result;
+});
+
+/**
+ * Sends an API message.
+ *
+ * @param {string} messageType - The message type.
+ * @param {string|Object} messageData - The message.
+ * @returns {void}
+ */
+function _sendApiMessage(messageType, messageData) {
+    let target;
+
+    // React native webview
+    // eslint-disable-next-line prefer-const
+    target = target || window.ReactNativeWebView;
```

Comment:
Is there a point to checking the truthiness of target?

Reply:
I wanted to be consistent with the (not yet existing) other checks, so then they can be in any order. Do you think I shouldn't have it here yet?